svm/src/account_loader.rs

```diff
@@ -134,6 +138,7 @@ pub(crate) fn load_accounts<CB: TransactionProcessingCallback>(
                         feature_set.is_active(&remove_rounding_in_fee_calculation::id()),
                     )
                 } else {
+                    *etx.1 = (Err(TransactionError::BlockhashNotFound), None, None);
```

Comment:
Is this necessary? Just wondering if there was an oversight in the previous code that's being fixed here. Or is this needed as part of this change.

Reply:
Nevermind, I see that similar change existed in now delete 'filter_executable_program_accounts`.